concordia/templates/admin/bulk_import.html

```diff
@@ -79,7 +79,7 @@ <h2>Import Tasks</h2>
             <h4>Messages</h4>
             <ul>
                 {% for message in messages %}
-                    <li class="message {% if message.level >= DEFAULT_MESSAGE_LEVELS.error %}message-error{% elif message.level >= DEFAULT_MESSAGE_LEVELS.warning %}message-warning{% endif %}">{{ message }}</li>
```

Comment:
This seems unrelated?

Reply:
It is, except it was cluttering up my console output.